doc/12-signal.md

```diff
@@ -0,0 +1,46 @@
+# Handling signals
```

Comment:
this document should be linked in the main readme

Reply:
Thx, ✅